CONTRIBUTING.md

```diff
@@ -142,4 +142,6 @@ function hello() {
 
 Be mindful that if you move some code in an example with highlighting, you also need to update the highlighting.
 
+Shobhit Singh
```

Comment:
This seems to be a mistake?

Reply:
I was following an article to make your first pull request and assumed this is done by all. Thanks for pointing out. 
Can you please also let me know if i need to open a PR again or It can be managed by you.

Reply:
> I was following an article to make your first pull request and assumed this is done by all. Thanks for pointing out.
> Can you please also let me know if i need to open a PR again or It can be managed by you.

You can make that change in your local, then commit and push in the same branch. It will update the same PR